python/cuml/test/test_pickle.py

```diff
@@ -29,14 +29,18 @@
 
 regression_models = {
     "LinearRegression": lambda: cuml.LinearRegression(),
+    "LogisticRegression": lambda: cuml.LogisticRegression(),
+    "LogisticRegressionNoInt": lambda: cuml.LogisticRegression(
+        fit_intercept=False),
```

Comment:
shouldn't a no intercept method be also added to the other linear models for consistency and future classes?

Reply:
LogisticRegression has slightly more complex intercept_ handling logic than other classes (pulling it out of qn which has its own special intercept support whereas for some other classes it's just a float), so I only worried about it here. Maybe it's still worth doing everywhere though... let me give it a little thought.

Reply:
I'll add fit_intercept checking for all regression models - thanks for pointing that out.